docs: Update screenshots (#265)
grafana/explore-profiles: docs/sources/choose-a-view/index.md

Similar to a `git diff`, it takes the selected flame graphs and highlights the differences between them.

![Diff flame graph](../images/explore-profiles-diff-flamegraph.png)

<!-- from old screenshot of cpu usage profile Red represents an increase in CPU usage from the baseline to the comparison and green represents a decrease. -->

### Favorites

> The **Favorites** view shows all your favorited visualizations. You can favorite a visualization by clicking on the star (⭐️) at its top-right corner.

Reviewer: ```suggestion
The **Favorites** view shows all your favorited visualizations. 
Clicking on the star (⭐️) at the top-right corner of a visualization saves it as a favorite.
```

> This feature helps you creating an overview of what's important, as well as offering the possibility to quickly jump to the **Flame graph** view or the **Labels** view.

Reviewer: ```suggestion
Using favories, you can create an overview of what's important, as well as  jump to the **Flame graph** view or the **Labels** view.
```
